packages/theme-chalk/src/common/var.scss

```diff
@@ -940,6 +940,10 @@ $rate: map.merge(
     'icon-size': 18px,
     'icon-margin': 6px,
     'icon-color': var(--el-text-color-placeholder),
+    'void-color': #c6d1de,
+    'star-color': #F7BA2A,
+    'disable-void-color': #EFF2F7,
+    'text-color': #1f2d3d,
```

Comment:
Let's unify it all in lowercase here. Or maybe you can lint theme by `pnpm format`.

Reply:
Thanks!